java/code/src/com/redhat/rhn/frontend/struts/test/RequestContextTest.java

```diff
@@ -38,12 +35,12 @@ public RequestContextTest(String name) {
     /**
      * @throws Exception if an error occurs
      */
-    public final void testGetLoggedInUser() throws Exception {
-        LoginActionTest ltest = new LoginActionTest();
-        HttpServletRequest request = ltest.loginUserIntoSessionTest();
-        RequestContext requestContext = new RequestContext(request);
-        assertNotNull(requestContext.getCurrentUser());
-    }
+//    public final void testGetLoggedInUser() throws Exception {
+//        LoginActionTest ltest = new LoginActionTest();
+//        HttpServletRequest request = ltest.loginUserIntoSessionTest();
+//        RequestContext requestContext = new RequestContext(request);
+//        assertNotNull(requestContext.getCurrentUser());
+//    }
```

Comment:
This commented out test seems to be equal to the previous one, remove it?

Reply:
Test adapted